Sources/protoc-gen-swift/main.swift

```diff
@@ -250,9 +250,11 @@ struct GeneratorPlugin {
   }
 
   private func sendReply(response: Google_Protobuf_Compiler_CodeGeneratorResponse) -> Bool {
+    // NOTE: I can only get rid of Data here if I use FileHandle.standardOutput.write(contentsOf:)
+    // below. This is only available in macOS 10.15.4 - should we increase min OS version?
```

Comment:
We don't currently list supported versions in `Package.swift`, but we do have things defined in the CocoaPods support. But even with that, this is sorta an interesting question, as there's no way in SwiftPM to express that one target has a different min version.  Since the plugin really only comes into play for the developers host platform, it would be nice to use a higher min OS version, but without a way to express that, we'd likely have to generally raise minimums to keep things happy.  Is there any way to _availability_ wrap this to also use the better API on linux when possible?

Reply:
I don't think getting rid of `Data` here is needed at all. This is the generator which can freely use `Data`. We only care about the _runtime_ where we want to make the bridging less expensive

Reply:
Yeah you're right Franz, we can just leave this here.